Composer/packages/extension-client/src/hooks/usePublishApi.ts

```diff
@@ -58,6 +58,13 @@ export function usePublishApi() {
   function userShouldProvideTokens(): boolean {
     return window[ComposerGlobalName].userShouldProvideTokens();
   }
+  function getPrimaryToken(): boolean {
```

Comment:
Does this ensure the token is not expired?

Reply:
this is not being used anymore, i removed it.